app/views/course_modules/show.html.erb

```diff
@@ -1,100 +1,138 @@
 <%= render "shared/components/desktop_layout" do %>
-<div class="box">
-  <%= link_to "Back", course_path(@course), class: "nav-link" %>
-</div>
-<div class="my-8 mx-4 text-right">
-  <% if policy(@course_module).edit? %>
-    <%= link_to edit_course_module_path(@course, @course_module), class: "btn-default" do %>
-      <span>Edit</span>
-    <% end %>
-  <% end %>
-  <% if policy(@course_module).destroy? %>
-    <%= link_to course_module_path(@course, @course_module), data: { turbo_method: :delete, turbo_confirm: "Are you sure want to delete?" }, class: "btn-default" do %>
-      <span>Delete</span>
-    <% end %>
-  <% end %>
-</div>
-
-<%= render @course_module %>
-
-<div class="box flex justify-between">
-  <h1 class="page-subtitle">Lessons</h1>
-  <div class="text-right">
-    <% if policy(@course_module).new? %>
-      <%= link_to new_course_module_lesson_path(@course, @course_module), class: "btn-default" do %>
-      <span>New lesson</span>
+  <div class="">
+    <div class="flex gap-4 items-center ">
+      <%= link_to course_path(@course), class: "nav-link" do %>
+        <span class="icon icon-arrow-left-gray icon-small"></span>
       <% end %>
-    <% end %>
-  </div>
-</div>
-<div class="box">
-  <ul id="lessons">
-    <% lessons_in_order(@course_module).each do |lesson| %>
-      <li class="text-sm border-b m-2" id="<%= dom_id lesson %>">
-        <%= link_to course_module_lesson_path(@course, @course_module, lesson) do %>
-          <div class="my-4">
-            <strong><%= lesson.title %></strong>
-            <% if policy(:lesson).moveup? %>
-              <p class="mb-2 text-gray-900 text-sm text-right"><%= link_to "Move Up", moveup_course_module_lesson_path(@course, @course_module, lesson), data: { turbo_method: :put }, class: "link" %></p>
-            <% end %>
-            <% if policy(:lesson).movedown? %>
-              <p class="mb-2 text-gray-900 text-sm text-right"><%= link_to "Move Down", movedown_course_module_lesson_path(@course, @course_module, lesson), data: { turbo_method: :put }, class: "link" %></p>
-            <% end %>
-          </div>
-          <div class="my-4">
-            <%= lesson.rich_description %>
-          </div>
+      <div class="font-medium text-sm">Back</div>
+    </div>
+    <%= render @course_module %>
+    <div class="box flex justify-between pt-6">
+      <h1 class="page-subtitle">Lessons</h1>
+      <div class="text-right">
+        <% if policy(@course_module).new? %>
+        <%= link_to new_course_module_lesson_path(@course, @course_module), class: "nav-link" do %>
```

Comment:
this is a block, block starts with `do` and end with `end` you should indent the blocks

Reply:
indentation corrected